app/views/assignments/edit/_review_strategy.html.erb

```diff
@@ -24,6 +24,11 @@
     <br/>
       <%= label_tag('assignment_form[assignment][max_reviews_per_submission]', 'Maximum number of reviews per submission:') %>
       <%= text_field_tag('assignment_form[assignment][max_reviews_per_submission]', @assignment_form.assignment.max_reviews_per_submission ||= 10, size: 1) %>
+      <br/>
+      <%= label_tag('assignment_form[assignment][num_reviews]', 'Set max number of reviews done by each student') %>
+      <span id='num_reviews_per_student_threshold'>
```

Comment:
Yes. It was used only in the instructor selected review option (same file) in the same file to set the review threshold. 

  <tr id="**instructor_selected_review_mapping_mechanism**" hidden='hidden'>
    <td>
      <%= form_tag :controller => 'review_mapping', :action => 'automatic_review_mapping', :id => @assignment_form.assignment.id do %>
        <input name="max_team_size" type="hidden" value="<%= @assignment_form.assignment.max_team_size %>"/>
        <%= radio_button_tag **'num_reviews'**, 'student', true %>
        <%= label_tag('num_reviews_per_student', **'Set number of reviews done by each student'**) %>